apps/site/lib/site_web/controllers/schedule/line_controller.ex

```diff
@@ -99,6 +99,7 @@ defmodule SiteWeb.ScheduleController.LineController do
     route_id
     |> services_by_route_id_fn.()
     |> dedup_services()
+    |> Enum.reject(&(&1.name == "Weekday (no school)"))
```

Comment:
Concerned about how fragile this appears to be — do we have a guarantee that no-school services will always have this exact name?

Reply:
Pretty sure I did confirm that with... @handorff maybe?

Reply:
I suppose we are already relying on a `/school/` regex match against the freeform `time_desc` text field on route patterns, so this isn't much _more_ fragile than the existing code.

Reply:
Perhaps instead of matching on the exact name, we could take the same approach here and match on `/school/`? That way if the word shows up in a slightly different arrangement, this will still work.

Reply:
> I suppose we are already relying on a `/school/ `regex match against the freeform `time_desc` text field on route patterns, so this isn't much more fragile than the existing code.

Yeah, that's basically why I felt ok with this change the way it is. Also hopefully in the future we will be [annotating no-school trips](https://zeroheight.com/2fedee66c/p/44c24d/b/255e95/t/92b70f) in a way that doesn't rely on arbitrary text matching anyhow.

I think if choosing between the two ways to match, I'd rather reject too few services while looking for `Weekday (no school)` rather than risk rejecting too many other services that happens to match `/school/`. 🤷‍♀

Reply:
> Also hopefully in the future we will be [annotating no-school trips](https://zeroheight.com/2fedee66c/p/44c24d/b/255e95/t/92b70f) in a way that doesn't rely on arbitrary text matching anyhow.

The approach described in the link is the one being implemented here, based on name matching. Did you mean something different?